cdap-ui/app/directives/datetime-picker/datetime.html

```diff
@@ -0,0 +1,55 @@
+<!--
+  Copyright © 2016 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<div class="datetime-range">
+  <div>
+    <uib-datepicker
+      ng-model="DatetimeController.date"
+      datepicker-options="DatetimeController.options"
+      ></uib-datepicker>
```

Comment:
looks strange but i don't know how you usually handle this. should the > be at the end of line 21?